src/app/api/auth/[...nextauth]/route.ts

```diff
@@ -0,0 +1,23 @@
+import NextAuth, { AuthOptions } from "next-auth";
+import KeycloakProvider from "next-auth/providers/keycloak"
+export const authOptions: AuthOptions = {
+  providers: [
+    KeycloakProvider({
+      jwks_endpoint: `${process.env.NEXT_CONTAINER_KEYCLOAK_ENDPOINT}/realms/shlp/protocol/openid-connect/certs`,
+      wellKnown: undefined,
+      clientId: process.env.KEYCLOAK_CLIENT_ID,
+      clientSecret: process.env.KEYCLOAK_CLIENT_SECRET,
+      issuer: process.env.KEYCLOAK_ISSUER,
+      authorization: {
+        params: {
+          scope: "openid email profile",
+        },
+        url: `${process.env.NEXT_LOCAL_KEYCLOAK_URL}/realms/shlp/protocol/openid-connect/auth`,
+      },
+      token: `${process.env.NEXT_CONTAINER_KEYCLOAK_ENDPOINT}/realms/shlp/protocol/openid-connect/token`,
+      userinfo: `${process.env.NEXT_CONTAINER_KEYCLOAK_ENDPOINT}/realms/shlp/protocol/openid-connect/userinfo`,
+    })
+  ]
+}
+const handler = NextAuth(authOptions);
+export { handler as GET, handler as POST }
```

Comment:
Similar comment around adding in some line separators. This is just to make the code a bit more readable